kafka-cruise-control-start.sh

```diff
@@ -184,6 +184,9 @@ fi
 # If Cygwin is detected, classpath is converted to Windows format.
 (( CYGWIN )) && CLASSPATH=$(cygpath --path --mixed "${CLASSPATH}")
 
+# Configure the JMX exporter agent.
+KAFKA_OPTS="$KAFKA_OPTS -javaagent:observability/jmx_prometheus_javaagent-1.5.0.jar=7071:observability/jmx_exporter_config.yaml"
```

Comment:
If https://github.com/linkedin/cruise-control/pull/2349 gets merged first, we should rename this `KAFKA_OPTS` to `JVM_OPTS`.